src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { Button } from "./Button";
+import { formatDateToNow } from "../lib/utils";
+import { Link } from "react-router-dom";
+
+const ArticleCard = (props) => {
+  const {
+    author,
+    title,
+    article_id,
+    topic,
+    created_at,
+    votes,
+    article_img_url,
+    comment_count,
+  } = props.articleData;
+
+  return (
+    <div className="border-t-2  ">
+      <Link>
+        <div className="hover:bg-light_gray p-2 hover:rounded-xl ">
+          <div className="flex items-center gap-3">
+            <p className="text-sm font-semibold text-gray-700">/{topic}</p>
+
+            <p className="text-sm text-gray-700">{author}</p>
+            <span className="text-xs font-extralight text-gray-400">•</span>
+            <p className="text-sm text-gray-400 ">
+              {formatDateToNow(created_at)}
+            </p>
+          </div>
+
+          <h2 className="mt-1 text-xl font-semibold text-gray-800">{title}</h2>
+
+          <div className="mt-2">
+            <img
+              src={article_img_url}
+              className="h-full w-full rounded-xl object-cover "
+            />
+          </div>
+
+          <div className="mt-2 flex items-center gap-4">
+            {/* vote button */}
```

Comment:
We should not have comments in our code, if you feel this is needed to add context it might be time to abstract this into it's own component